tests/timezones/tests.py

```diff
@@ -578,6 +578,38 @@ def test_write_datetime(self):
         fake_dt = datetime.datetime(2011, 9, 1, 17, 20, 30, tzinfo=UTC)
         self.assertEqual(event.dt, fake_dt)
 
+    def test_query_datetimes(self):
```

Comment:
This is not a proper place and not a proper set of tests. IMO we should add test(s) similar to the

```db_functions.datetime.test_extract_trunc.DateFunctionWithTimeZoneTests.test_extract_func_with_timezone```

where we can set `tzname` directly  (with the `TIME_ZONE != 'UTC'` setting).